### ISSUE TITLE
[BUG] Top Of Descent (Issues)

### ISSUE THREAD

**Issue by Snapmatics** [flybywiresim/a32nx#502]
**Mod Version**

Latest Master (0.2.0)

**Describe the Bug**

- There are so many **T/D** icons on Departure/before the Approach.
- Also, the icon of **T/D** should be ON the line and not offset.

**To Reproduce**

1. Good/Accurate calculations for the **T/D**.
2. Make it so the **T/D** is ON the course/green line instead of being offset.
3. Make the **T/D** not showing all the time while you are departing/cruising/before approaching.

**Expected Behavior**

- We must have a **T/D** before we start descending and then we must have another **T/D** that shows when our airplane will reach our set ALT on the ND.

**Actual Behavior**

- Because I have so many **T/D** before I should start my descent, I end up burning so much fuel by flying very low and having more than 100 miles to reach to the airport (100 miles is an example).

**References**

![TD Issues](https://user-images.githubusercontent.com/43104362/92434141-c24ed480-f1a7-11ea-9358-71d08d6873e4.png)

**Additional Context**

None

**Comment by ThePilot916** [comment]
#436 similar
@Snapmatics could you attach an image of your CDU along with?

**Comment by Snapmatics** [comment]
> #436 similar
> @Snapmatics could you attach an image of your CDU along with?

Well, I ended that flight so I will have to start a new one. It seems like it happens all the time for my friends as well.
I will for sure attach a new image when I am flying again. 

BUT, I remember I didn't had any T/D on my CDU but only the DESCE waypoint.

**Comment by nabil74** [comment]
Previously raised here also
Same issue

https://github.com/flybywiresim/a32nx/issues/436#issue-694221059

**Comment by MisterChocker** [comment]
duplicate of #436

**Comment by Snapmatics** [comment]
> duplicate of #436

Alright :)
